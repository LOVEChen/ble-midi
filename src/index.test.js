/* eslint-env jest */

test('constants', () => {
  const lib = require('./index');
  expect(lib).toMatchSnapshot();
});
