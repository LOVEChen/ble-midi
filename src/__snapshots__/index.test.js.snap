// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants 1`] = `
Object {
  "createPeripheral": [Function],
  "listDevices": [Function],
}
`;
